app/views/dashboard/student.haml

```diff
@@ -8,32 +8,26 @@
       %li Your #{interview.application_submission.department.name} interview
       has been scheduled for #{interview.information}.
     .interview_review_link
-      = link_to 'Review your application', interview.application_submission
+      = link_to 'View your application', interview.application_submission
 %h2 Available Applications
 Please select the application which you would like to submit.
 - @positions.each do |department, positions|
   %h3= department.name
   - positions.each do |position|
-    - app_submitted = @application_submissions.try(:keys).try(:include?, position)
-    - if app_submitted
-      // most recent application, if more than one
-      - application = @application_submissions[position].last
-      - should_show = application.pending? || should_show_denied_applications?
-    - if app_submitted && should_show
-      %li= link_to "Review your application for #{position.name}", application
-    - if position.application_template.try(:active?)
-      - if app_submitted
-        - if should_allow_resubmission?
-          -if application.reviewed
+    // most recent application, if more than one
+    - if application = @application_submissions[position].try(:last)
```

Comment:
`@application_submissions` can be `nil` if the person hasn't applied yet (no `@current_user`).

Also, Rubocop would like you to put `()`s around this assignment to signal that you totally didn't mean `==`